src/ImageSharp.Formats.Gif/GifDecoderCore.cs

```diff
@@ -62,6 +68,15 @@ internal class GifDecoderCore<TColor>
         private GifGraphicsControlExtension graphicsControlExtension;
 
         /// <summary>
+        /// Initializes a new instance of the <see cref="GifDecoderCore{TColor}"/> class.
+        /// </summary>
+        /// <param name="options">The decoder options.</param>
+        public GifDecoderCore(IGifDecoderOptions options)
+        {
+            this.options = options ?? new GifDecoderOptions();
```

Comment:
This will happen when you call the constructor of Image without options (options=null). The GifDecoder will just forward the value.